workflowproj/go.mod

```diff
@@ -39,6 +35,7 @@ require (
 	github.com/google/uuid v1.3.0 // indirect
 	github.com/josharian/intern v1.0.0 // indirect
 	github.com/json-iterator/go v1.1.12 // indirect
+	github.com/kiegroup/kogito-serverless-operator/api v0.0.0-20230609141400-51026c39778d // indirect
```

Comment:
that means when branching/releasing, we need to update that reference right ?

Reply:
In this process, replace here with the latest tag, and then do a go mod download/tidy.